lib/matplotlib/tests/test_axes.py

```diff
@@ -204,6 +204,28 @@ def test_formatter_ticker():
     ax.autoscale_view()
 
 
+def test_auto_formatter():
```

Comment:
can you break this into two tests, one for the string functionality and one for the function functionality?

Reply:
Done